src/components/paragraphs/sup-file-list/file-list-paragraph.tsx

```diff
@@ -28,17 +29,17 @@ const FileListParagraph = ({paragraph, ...props}: Props) => {
   }
 
   return (
-    <div {...props}>
+    <div className={twMerge("centered max-w-800", props.className)} {...props}>
```

Comment:
put the `{...props}` before `className`. Otherwise, the className in the props will override your twMerge
```suggestion
    <div  {...props} className={twMerge("centered max-w-800", props.className)}>
```

Reply:
Thanks for catching this, Mike! I knew I was missing something 😮‍💨 

![image](https://github.com/user-attachments/assets/7e4aee19-a567-4ec6-ba14-8bb14f6ece23)